src/5_1.py

```diff
@@ -0,0 +1,39 @@
+"""Оформите решение задач из прошлых домашних
+работ в функции. Напишите функцию runner.
+(все станет проще когда мы изучим модули,
+getattr и setattr)
+runner() – все фукнции вызываются по очереди
+runner(‘func_name’) – вызывается только
+функцию func_name.
+runner(‘func’, ‘func1’...) - вызывает все
+переданные функции """
+
+from src import homeworks
+
+parameters = {"hi_name": ["Ivan", "Ivanov"]}
+
+
+def runner(*func):
+    db = []
+    [db.append(element) for element in dir(homeworks) if element[:2] != "__"]
+
+    try:
+        for element in func:
+            callable(getattr(homeworks, element)(parameters.get(element)))
```

Comment:
your code first call elem and next check if it's callable. https://www.geeksforgeeks.org/callable-in-python/

Reply:
p3 from previous review not addressed